pkg/tfvars/baremetal/baremetal.go

```diff
@@ -86,6 +86,7 @@ func TFVars(numControlPlaneReplicas int64, libvirtURI, apiVIP, imageCacheIP, boo
 		driverInfo := accessDetails.DriverInfo(credentials)
 		driverInfo["deploy_kernel"] = fmt.Sprintf("http://%s/images/ironic-python-agent.kernel", net.JoinHostPort(imageCacheIP, "80"))
 		driverInfo["deploy_ramdisk"] = fmt.Sprintf("http://%s/images/ironic-python-agent.initramfs", net.JoinHostPort(imageCacheIP, "80"))
```

Comment:
Actually, it looks like I was wrong about this.

The [`get_live_images.sh` script](https://github.com/openshift/ironic-rhcos-downloader/blob/master/get-live-images.sh) in ironic-rhcos-downloader puts the kernel/ramdisk (alongside the rootfs) in the same place, so these URLs will still work.

The ironic configuration script [automatically adds a kernel arg](https://github.com/metal3-io/ironic-image/blob/master/scripts/coreos-ipa-common.sh#L7-L19) (for PXE only) to tell CoreOS where to find the rootfs - we decided to do this instead of concatenating.